docs/access.md

```diff
@@ -0,0 +1,367 @@
+Access control for groups and group content
+===========================================
+
+Controlling access to groups and group content is one of the most important
+aspects of building a group based project. Having separate access rules for
+different groups and the content in them is more complex than the standard role
+and permission based access system built into Drupal core so Organic Groups has
+extended the core functionality to make it more flexible and, indeed, _organic_
+for developers to design their access control systems.
+
+Group level permissions
+-----------------------
+
+The first line of defense is group level access control.
+Group level permissions apply to the group as a whole. OG ships with a number
+of permissions that control basic group and membership management tasks such as:
+- Administration permissions such as `update group`, `delete group`, `manage
+  members` and `approve and deny subscription`.
+- Membership permissions such as `subscribe` and `subscribe without approval`
+  which can be granted to non-members.
+
+Developers can define their own group level permissions by implementing an event
+listener that subscribes to the `og.permission` event and instantiating
```

Comment:
Normally I'd say let's use the interface but it seems like we chose the EVENT_NAME poorly:
```suggestion
listener that subscribes to the `PermissionEventInterface::EVENT_NAME` event and instantiating
```